ethexe/runtime/common/src/journal.rs

```diff
@@ -133,8 +137,7 @@ impl<S: Storage> JournalHandler for NativeJournalHandler<'_, S> {
     }
 
     fn gas_burned(&mut self, _message_id: MessageId, _amount: u64) {
-        // TODO
-        // unreachable!("Must not be called here")
+        unreachable!("Handled inside runtime by `RuntimeJournalHandler`")
```

Comment:
actually it will be reachable here, because we also need to reduce exec balance, not only decrease allowance - but it could be handled in runtime as well

Reply:
Added TODO